src/ut/call_list_store_test.cpp

```diff
@@ -81,10 +105,22 @@ class CallListStoreFixture : public ::testing::Test
   {
     _store.stop();
     _store.wait_stopped();
+    delete _iter; _iter = NULL;
+  }
+
+  AddrInfo create_target(std::string address)
```

Comment:
This is duplicated in cedar, and doesn't need any class info - could we make it a common function in cpp-common?